src/Elcodi/Component/Product/Entity/Variant.php

```diff
@@ -80,6 +80,16 @@ class Variant implements VariantInterface
     protected $options;
 
     /**
+     * Gets name
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return $this->product->getName();
```

Comment:
The variant's name is the product's name? Then the method should be called `getProductName`